src/swap/components/Swap.tsx

```diff
@@ -21,6 +21,7 @@ export function Swap({
   onError,
   onStatus,
   onSuccess,
+  isSponsored = false,
```

Comment:
nit: alphabetic

Reply:
done - this was from when we called it `sponsored`